### PR TITLE
feat: repository path template

### PR DESCRIPTION
**Pull request by mattia-fumo**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/console/company-configuration/providers/configure-provider.mdx [hunk 1]
```diff
@@ -64,6 +64,10 @@ In this step, you can insert some general details about your provider:
 * **Base URL** (*required*): the reference endpoint URL for the provider;
 * **Description**: a brief description of the provider.
 
+<a name="repositoryPathTemplate"></a>
+
+If in [Step 1](#step-1-provider-services) the **Git Provider** capability has been selected, an additional **Repository path template** field will appear in this step, this field controls where project repositories, created using this provider, will be located. It can also contain a special interpolation variable `{{projectId}}` that will be replaced with the actual project identifier during the [project creation](/console/project-configuration/create-a-project.mdx) phase.
```
Comment (RiccardoCorona-MiaPlatform) [inline docs/console/company-configuration/providers/configure-provider.mdx:69]:
```suggestion
If in [Step 1](#step-1-provider-services) the **Git Provider** capability has been selected, an additional **Repository path template** field will appear in this step. This field controls where Project repositories, created using this provider, will be located, and can also contain a special interpolation variable `{{projectId}}` that will be replaced with the actual Project identifier during the [Project creation](/console/project-configuration/create-a-project.mdx) phase.
```

File: docs/console/project-configuration/create-a-project.mdx [hunk 2]
```diff
@@ -59,24 +59,26 @@ The Project creation process is divided into three phases:
 
     If there are no configured Git Providers for the company, an informative modal is displayed, and you will not be able to continue.
 
-   * **Git repo path**: the path, calculated from Project name and company, where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
+   * **Git repo path**: the path, where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
```
Comment (RiccardoCorona-MiaPlatform) [inline docs/console/project-configuration/create-a-project.mdx:62]:
```suggestion
   * **Git repo path**: the path where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
```

File: docs/console/project-configuration/create-a-project.mdx [hunk 3]
```diff
@@ -59,24 +59,26 @@ The Project creation process is divided into three phases:
...
+
+    This field will be precompiled with the [**Repository path template**](/console/company-configuration/providers/configure-provider.mdx#repositoryPathTemplate) of the chosen provider, if any, interpolated with the project identifier.
```
Comment (RiccardoCorona-MiaPlatform) [inline docs/console/project-configuration/create-a-project.mdx:64]:
```suggestion
    This field will be precompiled with the [**Repository path template**](/console/company-configuration/providers/configure-provider.mdx#repositoryPathTemplate) of the chosen provider, if any, interpolated with the Project identifier.
```

File: docs/console/project-configuration/create-a-project.mdx [hunk 4]
```diff
@@ -59,24 +59,26 @@ The Project creation process is divided into three phases:
...
 
     :::caution
     Make sure the selected Git Provider has the right permissions for creating projects on the configured path.
```
Comment (RiccardoCorona-MiaPlatform) [inline docs/console/project-configuration/create-a-project.mdx:67]:
```suggestion
    Make sure the selected Git Provider has the right permissions for creating Projects on the configured path.
```

**Review by RiccardoCorona-MiaPlatform — APPROVED** [review]

File: docs/console/project-configuration/create-a-project.mdx [hunk 5]
```diff
@@ -59,24 +59,26 @@ The Project creation process is divided into three phases:
 
     If there are no configured Git Providers for the company, an informative modal is displayed, and you will not be able to continue.
 
-   * **Git repo path**: the path, calculated from Project name and company, where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
+   * **Git repo path**: the path where the configuration will be saved on the Git Provider. It specifies the location of your Project on the Git Provider website;
```
Comment (Danielecina) [inline docs/console/project-configuration/create-a-project.mdx:62]: è corretto il nome? lato FE non mostriamo Repository path template?

Comment (mattia-fumo) [reply docs/console/project-configuration/create-a-project.mdx:62]: `Git repo path` is the correct name  for the field. Maybe you are confusing it with the one on the provider form.

**Review by Danielecina — APPROVED** [review]

**Pull request merged**